### PR TITLE
Class-a, ocdb.stanza, gen-ocdb-app, some proj files

### PR DESCRIPTION
**Pull request by ariel-hi**
`class-a.stanza` : Dummy landpattern + symbol to allow this file to be exported.
`ocdb.stanza` : Cleaned up, includes more files. Removed just a few.
`backplanes.stanza` : Removed. Let's define a location for old files that we're interested in keeping.
`gen-ocdb-app.stanza` : Updated package imports. I haven't ran the generator.

### CODE REVIEW

File: scripts/gen-ocdb-app.stanza [hunk 1]
```diff
@@ -6,14 +6,13 @@ defpackage fig-app :
...
-  import esir/repl-lib
-  import ocdb/tests/default-harness
+  import jitx
+  import jitx/gen
+  import jitx/commands
+  import ocdb/defaults
   import ocdb/land-patterns
   import ocdb/generic-components
-  import jitpcb/visualizer
+  import jitx/visualizer
```
Comment (PhilippeFerreiraDeSousa) [inline scripts/gen-ocdb-app.stanza:15]: Have you run this? Do you get shiny svgs?

Comment (ariel-hi) [reply scripts/gen-ocdb-app.stanza:15]: That's the next step. Just modified the imports for now.

**Review by PhilippeFerreiraDeSousa — APPROVED** [review]

**Review by d-haldane — COMMENTED** [review]
Instead of the dummy landpattern, just use the generator from pin-header, in utils/generic-components

**Comment by ariel-hi** [comment]
@d-haldane I updated power-connector to use pin-header-pkg and box-symbol generators.

**Review by d-haldane — APPROVED** [review]

**Pull request merged**